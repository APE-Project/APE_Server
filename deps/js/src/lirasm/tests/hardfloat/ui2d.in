i = immi -1
d = ui2d i

retd d

