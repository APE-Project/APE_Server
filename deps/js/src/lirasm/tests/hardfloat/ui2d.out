Output is: 4.29497e+09
